wp-content/themes/csisnuclearnetwork/assets/_scss/pages/archive.scss

```diff
@@ -103,129 +103,95 @@
 
   &__base {
     @include shadow('lg');
-
-    @include breakpoint('small') {
-      margin-top: rem(32);
-    }
-
-    @include breakpoint('medium') {
-      margin-top: rem(40);
-    }
-
-    @include breakpoint('large') {
-      margin-top: rem(56);
-    }
+    margin-top: rem(32);
   }
 }
 
-.pagination {
-  &,
-  &__results {
-    @extend %text-style-label-medium;
-    font-weight: normal;
-  }
-
-  &__results {
-    margin-bottom: rem(40);
-    padding-bottom: rem(8);
-    color: $color-black-190;
-    border-bottom: 1px solid $color-border-dark-130;
+.pagination__results {
+  @extend %text-style-label-medium;
+  padding-bottom: rem(16);
+  color: $color-black-180;
+  font-weight: normal;
+  border-bottom: 1px solid $color-border-dark-130;
+}
 
-    @include breakpoint('large') {
-      padding-bottom: rem(16);
-    }
-  }
+.pagination__wrapper {
+  display: flex;
+  gap: rem(16);
+  justify-content: flex-start;
+  align-items: center;
+  color: $color-black-165;
 
-  .nav-links {
-    display: flex;
+  @include breakpoint('small') {
     gap: rem(20);
-    justify-content: flex-end;
-    align-items: center;
-    color: $color-black-165;
+  }
 
-    @include breakpoint('small') {
-      gap: rem(24);
-    }
+  @include breakpoint('medium') {
+    gap: rem(24);
+  }
 
-    @include breakpoint('medium') {
-      gap: rem(32);
-    }
+  .pagination {
+    @extend %text-style-label-medium;
+    font-weight: normal;
   }
 
   .current {
     position: relative;
-    color: $color-black-100;
+    color: $color-white-190;
+    background: $color-accent-primary-blue-300;
 
-    &::before {
-      content: '';
-      position: absolute;
-      bottom: 100%;
-      left: 0;
-      display: block;
-      width: 100%;
-      height: 4px;
-      background: $color-bg-dark-200;
+    &:hover {
+      color: $color-white-190 !important;
+      background: $color-accent-primary-blue-300 !important;
```

Comment:
If you move `.current` below `.page-numbers`, you don't have to use `!important`